### PR TITLE
disable auth0 for local dev

### PR DESCRIPTION
**Pull request by spasovski**
Not sure if this is the best approach but it worked locally.

### CODE REVIEW

File: backend/settings/local.py [hunk 1]
```diff
@@ -2,3 +2,4 @@
 
 
 MIDDLEWARE.append('backend.middleware.dev_cors_middleware')  # noqa
+REST_FRAMEWORK['DEFAULT_PERMISSION_CLASSES'] = ('rest_framework.permissions.AllowAny',) # noqa
```
Comment (robhudson) [inline backend/settings/local.py]: Maybe add a comment here to comment this out if we want to test auth0 locally?

**Comment by robhudson** [comment]
The tests depend on auth0 being active. To get this to work on circleci you can do the following:

- Add `DJANGO_SETTINGS_MODULE=backend.settings.production` to `backend.env.dist`

That's what CircleCI uses for the env settings. That may or may not be all we need, not sure, b/c I notice the auth0 stuff (id/secret) are empty.

Another option... Put the `AllowAny` thing in the local settings but commented out and when we want to bypass it we uncomment it but never commit it to git.

**Comment by spasovski** [comment]
Makes me wonder if there is a way to do this via env vars. I get that we should have the ability to test with auth locally.

**Comment by spasovski** [comment]
No longer matters.

**Pull request closed without merging**